pkg/cmd/pulumi/packagecmd/package_add.go

```diff
@@ -28,6 +28,59 @@ import (
 	"github.com/spf13/cobra"
 )
 
+// InstallPackage installs a package to the project by generating an SDK and linking it.
+// It returns the path to the installed package.
+func InstallPackage(ws pkgWorkspace.Context, pctx *plugin.Context, language, root,
+	schemaSource string, parameters []string,
+) error {
+	pkg, err := SchemaFromSchemaSource(pctx, schemaSource, parameters)
+	if err != nil {
+		var diagErr hcl.Diagnostics
+		if errors.As(err, &diagErr) {
+			return fmt.Errorf("failed to get schema. Diagnostics: %w", errors.Join(diagErr.Errs()...))
+		}
+		return fmt.Errorf("failed to get schema: %w", err)
+	}
+
+	tempOut, err := os.MkdirTemp("", "pulumi-package-")
+	if err != nil {
+		return fmt.Errorf("failed to create temporary directory: %w", err)
+	}
+	defer os.RemoveAll(tempOut)
+
+	local := true
+
+	err = GenSDK(
+		language,
+		tempOut,
+		pkg,
+		"",    /*overlays*/
+		local, /*local*/
+	)
+	if err != nil {
+		return fmt.Errorf("failed to generate SDK: %w", err)
+	}
+
+	out := filepath.Join(root, "sdks")
+	err = os.MkdirAll(out, 0o755)
+	if err != nil {
+		return fmt.Errorf("failed to create directory for SDK: %w", err)
+	}
+
+	outName := pkg.Name
+	if pkg.Namespace != "" {
+		outName = pkg.Namespace + "-" + outName
+	}
+	out = filepath.Join(out, outName)
+	err = CopyAll(out, filepath.Join(tempOut, language))
```

Comment:
We should make sure the destination is empty. Otherwise we're going to leave old files in the SDK and just overwrite anything thats in both old and new.

Reply:
`package add` doesn't do that, so `package install` shouldn't either? Or rather, maybe change both in a separate PR?

Reply:
Yeh both should do it